### DISCUSSION TITLE
Reset not clearing error message

### DISCUSSION THREAD

**Discussion by vkwd** [community/community#21463] [Other Feature Feedback, Questions, & Ideas]
<p>As a novice I have created a calculator at <a href="https://vkwd.github.io/DRcalc/" rel="noopener nofollow ugc">https://vkwd.github.io/DRcalc/</a>.<br>
in javascript/html/css.<br>
on inputting data in ‘Basic Pension’ " below 3500 and above 99999" , when a user click ‘calculate’  it gives rightly a error  message  " please enter a valid basic pension…" .<br>
However when I click reset it restores the error  data to 0, but  error message above  is not cleared.<br>
I have tried all that I could lay hand to like adding event listener on click on reset / and changing validating code etc. but no success.<br>
Please advise me how this can be achieved by providing example if possible.<br>
thanks</p>

**Comment by fire-eggs** [comment, marked as answer]
<p>An <code>&lt;input type="reset"&gt;</code> function is going to clear only input fields.</p>
<p>So you could make the error message field an input / readonly field.</p>
<p>Alternatively, drop the reset capability off the reset button, add an <code>onClick</code> like you have for the calculate button, and in the function you create, clear the input fields and the text field.</p>
